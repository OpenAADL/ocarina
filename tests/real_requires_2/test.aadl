package BugTest
public
system A
end A;

system implementation A.impl
annex real_specification {**
    theorem test
    foreach s in system_set do
        requires (sub1);
        check (1 = 1);
    end test;
**};
end A.impl;

end BugTest;
