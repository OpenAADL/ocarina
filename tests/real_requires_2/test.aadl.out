theorems.real:9:10 Backends: error : Property is false for instance 6 (a.impl)
theorems.real:1:01 Backends: error : requirements of sub1 are not fulfilled
test.aadl:8:05 Backends: error : requirements are not fulfilled for theorem test
test execution
Processing requirement : sub1
Processing requirement : sub2
-------------------------------------
Evaluating theorem sub2

 * Iterate for variable: a.impl
 => Result: FALSE

theorem sub2 is: FALSE

-------------------------------------
Evaluating theorem sub1

 * Iterate for variable: a.impl
 => Result: TRUE

theorem sub1 is: FALSE


theorem test is: FALSE

